tasks/facial_landmark_detection/README.md

```diff
@@ -0,0 +1,60 @@
+# Facial Landmarks
+
+This project demonstrates the use of Archai to find efficient neural architecture for facial landmarks localization. We follow the general methodology described in the papers [1] and [2], which use a synthetic face dataset and MobileNet V2 as the starting backbone.
+
+## Download the Dataset
+
+The synthetic face dataset can be downloaded from the [FaceSynthetics repository](https://github.com/microsoft/FaceSynthetics).
+
+## Perform Neural Architecture Search
+
+The default parameters in the `search_config.yaml` file search models with 1/10 of the dataset and 30 epochs. These parameters can be adjusted to suit different needs.
+
+To perform the neural architecture search, run the following command:
+
+```bash
+python search.py --config search_config.yaml --output_dir . --data_dir face_synthetics/dataset_100000
+```
+
+Note that the paths need to be modified to match what is on your system. 
+
```

Comment:
Would be cool if you could add a paragraph on how you designed the search space.   For example, Piero wrote this which I liked: https://github.com/microsoft/archai/tree/main/tasks/face_segmentation#search-space, but I'd like even more than this - what intuition did you have that caused you to parametrize the things you did, like channel_multiplier and depth_multiplier, and how did you come up with the discrete set of arch_defs ?  I really think this is the meat of Archai documentation that is needed, since this is the key to having an effective search.

Reply:
You are absolutely right. Awesome suggestion!